_posts/en/newsletters/2024-06-28-newsletter.md

```diff
@@ -0,0 +1,223 @@
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #29575][] simplifies the peer misbehavior scoring system to only
+  use two increments: 100 points (results in immediate disconnection and
+  discouragement) and 0 points (allowed behavior). Most types of misbehaviors
+  have been bumped to a score of 100, while 2 behaviors that are considered
+  unavoidable have been reduced to 0. This PR also removes the 8-header limit
+  heuristic for determining potential [BIP130][] announcements, and now treats
+  all non-connecting headers as potential announcements.
+
+- [Bitcoin Core #28984][] adds support for a limited version of [package][topic
+  package relay] [replace-by-fee][topic rbf] with clusters of size two including
+  [Topologically Restricted Until Confirmation (TRUC)][topic v3 transaction
+  relay] transactions (aka v3 transactions). These clusters can only replace an
+  existing cluster of the same size or smaller. See [Newsletter #296][news296
+  packagerbf] for related context. <!-- there’s no specific mention in the PR that TRUC transactions are supported for package RBF but from my understanding, and from what was said in Newsletter #296, it is the case?-->    
+
+- [Core Lightning #7388][] removes the ability to create non-zero-fee
+  [anchor-style][topic anchor outputs] channels to conform to recent changes in
+  the BOLT specification, while maintaining support for existing channels. Core
+  Lightning was the only implementation to have this, and only in experimental
+  mode. Other updates include rejecting `encrypted_recipient_data` containing
+  both `scid` and `node`, adding LaTeX formatting to the onion specification,
+  and other BOLT specification changes mentioned in Newsletters [#259][news259
+  bolts] and [#305][news305 bolts].
+
+- [LND #8734][] improves the payment route estimation abort process when a user
+  interrupts the `lncli estimateroutefee` RPC command by making the payment loop
+  aware of the client's streaming context. Previously, interrupting this command
+  would cause the server to continue [payment probing][topic payment probes]
+  routes unnecessarily. See Newsletter [#293][news293 routefee] for a previous
+  reference to this command. 
+
+- [LDK #3127][] implements non-strict forwarding to improve payment reliability,
+  allowing [Hash Time Locked Contract (HTLCs)][topic htlc] to be forwarded to a
```

Comment:
@Gustavojfe FYI: we don't need to expand abbreviations for which we have a topic page; it's ok to just link to them instead (e.g. `[HTLC][topic htlc]`).  TRUC, used above, is still a bit new, so I'm leaving that in, but I think we can stop expanding that in future newsletters.